DEVELOP.rst

```diff
@@ -91,6 +91,18 @@ The list of the operations are written in the subfolder
         "__openerp__.py": "__manifest__.py",
     }
 
+* Warnings in files. For exemple, for migration from version 12.0 to 13.0
```

Comment:
You are free to fix spelling in new/modified lines :)

Reply:
Changed in new push